internal/config/config.go

```diff
@@ -88,4 +93,8 @@ func loadFromEnv() {
 	} else if githubToken := os.Getenv("GITHUB_TOKEN"); githubToken != "" {
 		Av.GitHub.Token = githubToken
 	}
+
+	if apiToken := os.Getenv("AVIATOR_APITOKEN"); apiToken != "" {
```

Comment:
Can we do 
```suggestion
	if apiToken := os.Getenv("AV_API_TOKEN"); apiToken != "" {
```
instead?